### PR TITLE
Add content-type. Document meaning of fields

### PR DESCRIPTION
**Pull request by gmoore**

### CODE REVIEW

File: content/admin/org_settings/integrations/GenericWebhook.md [hunk 1]
```diff
@@ -16,40 +16,46 @@ Contrast supports generic webhook integration, which allows you to receive notif
...
-* Vulnerability Rule
-* Vulnerability ID
-* Event Type
-
-If you were using the Generic Webhook to create an integration, the sample payload could be as follows:
+The placeholders available for use that will always return a value:
+
+* `$Title` - The title of the vulnerability
+* `$Message` - A human-readable summary of the event.
+* `$EventType` - The type of event which will be one of NEW_VULNERABILITY, NEW_ATTACK, ATTACK_UPDATE, NEW_ATTACK_APPLICATION, ATTACK_END, VULNERABILITY_CHANGESTATUS_OPEN, VULNERABILITY_CHANGESTATUS_CLOSED:, SERVER_OFFLINE, NEW_ASSET, EXPIRING_LICENSES, NEW_VULNERABILITY_COMMENT, ATTACK_EVENT_COMMENT.
```
Comment (contrast-product) [inline content/admin/org_settings/integrations/GenericWebhook.md]: Is the ":" required that's currently attached to "VULNERABILITY_CHANGESTATUS_CLOSED"?

Comment (gmoore) [reply content/admin/org_settings/integrations/GenericWebhook.md]: Nope. Good catch!

**Comment by contrast-product** [comment]
@gdvdsn-contrast, can we merge and deploy soon?

**Comment by gdvdsn-contrast** [comment]
@gmoore I'll get it published today!

**Pull request merged**